src/Maker/MakeResetPassword.php

```diff
@@ -54,12 +55,14 @@ class MakeResetPassword extends AbstractMaker
     private $fileManager;
     private $doctrineHelper;
     private $entityClassGenerator;
+    private $params;
 
-    public function __construct(FileManager $fileManager, DoctrineHelper $doctrineHelper, EntityClassGenerator $entityClassGenerator)
+    public function __construct(FileManager $fileManager, DoctrineHelper $doctrineHelper, EntityClassGenerator $entityClassGenerator, ParameterBagInterface $params)
```

Comment:
The changes to this maker are not needed. All of the templates already have a `$use_typed_properties` global variable available which is ultimately set by the https://github.com/symfony/maker-bundle/blob/313b5669a5370bf36cd34fa8f5b5bbbfa5fb8aa8/src/Util/PhpCompatUtil.php#L39 method.